setup.py

```diff
@@ -59,5 +65,5 @@ def make_distribution(self):
         "sql": sql_deps,
         "all": all_deps,
     },
-    python_requires=">=3.8",
+    python_requires=">=3.6",
```

Comment:
Should we exclude python 3.7?

Reply:
I don't see any point in excluding that - we just require `pandas==1.1.5` for `Python < 3.8`, and it should work.

Reply:
If it should work, why do we name compat files with `3.6` suffix? Won't a confusion be in future?

Also, if python >= 3.7.1 and < 3.8, why do we require pandas==1.1.5? For these python versions pandas>=1.2.0 and < 1.4.0 should work.